{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -32,6 +32,7 @@ APP_ALLOWED_HOSTS = ['0.0.0.0', 'localhost', '127.0.0.1']
 # ================
 # See https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/
 
+# TODO: Set
```

Comment:
The TODO is good 🚀 
Comment: In practice we are not setting this in the cfg file for two reasons:
- It might not be a good idea to put a password on a docker image that might somehow be downloaded by someone else, you need to take into account security of the image hub etc.
- It might actually change quite often, depending on rotation policies, etc. So re-building the image makes not much sense.

What we do is use (sort of) environment variables, which are much easier to change + re-deploy (re-load the app), which is much faster.


Not sure if this is worth putting here or we should start a "deployment tips" section in the docs.

Reply:
I like the deployment tips section idea. I am indeed not changing the secret key here but in our environment variables (well, via a .env file). It was more a disclaimer to remind the deployer to set it somewhere and you are right it might encourage the deployer to set it in place. I will be adding to the deployment documentation and see how things shake out then